src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandler.java

```diff
@@ -80,6 +82,13 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
         final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
         final SscsCaseData sscsCaseData = caseDetails.getCaseData();
 
+        if (isBundleAdditionWarning(sscsCaseData)) {
+            PreSubmitCallbackResponse<SscsCaseData> response =
+                new PreSubmitCallbackResponse<>(callback.getCaseDetails().getCaseData());
+            response.addWarning("Some documents in this Bundle contain the same addition letter. Are you sure you want to proceed?");
```

Comment:
After adding warning,  can be proceed further? its look like we are not proceeding further and its just return the response..

Reply:
Logic added for ignore warnings